### DISCUSSION TITLE
Packet Decryption (DPI) / MITM man in the middle

### DISCUSSION THREAD

**Discussion by silentbits** [Security-Onion-Solutions/securityonion#5084] [Unsupported Versions]
I'm looking to add DPI to SO.  It is understood this is not something that is integrated into SO.  The latest information I have seen is from Netresec from January 2020 but is also for a linux-based installation, and not the new docker implementation.

Can anyone help out?

**Comment by silentbits** [comment]
?

**Comment by weslambert** [comment]
You would need to already have this performed before the data is fed to Security Onion, or use something like PolarProxy as you mentioned.  As I get time, I'll look at working with Erik to get the documentation updated for the latest version of Security Onion.

**Comment by silentbits** [comment]
Can the documentation include how to install / configure Polarproxy with the latest OS (docker) version as the previous documentation did not really cover this and we are not familiar enough how to configure docker within SO.

Also, would you be able to answer our question regarding adding a second interface? Our needs require separation (vlans) of management and log traffic (importing to SO). 

https://github.com/Security-Onion-Solutions/securityonion/discussions/5003
